src/components/ResetPasswordForm.js

```diff
@@ -0,0 +1,188 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import styles from '../styles/entry.module.css';
+import axios from 'axios';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { connect } from 'react-redux';
+import { restPasswordAction, closeMessage } from '../redux/actions/resetPasswordAction';
+import { LinearProgress } from '@material-ui/core';
+import translate from '../languages/translate';
+import { BrowserRouter, Link, Redirect } from 'react-router-dom';
+import SocialButtons from './SocialButtons';
+import SvgMap from './SvgMap';
+import LanguageButtons from './LanguageButtons';
+import { withRouter } from 'react-router-dom';
+
+class ResetPasswordForm extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			password: '',
+			passwordErrorStatus: false,
+			confirmPassword: '',
+			confirmPasswordErrorStatus: false,
+			open: false,
+			errorMessage: ''
+		};
+	}
+	validate = () => {
+		let isError = false;
+		const errors = {
+			passwordErrorStatus: false,
+			confirmPasswordErrorStatus: false
+		};
+		const regex = RegExp(/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[a-zA-Z\d@$.!%*#?&]/);
+		// const results = this.state.password.match(/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[a-zA-Z\d@$.!%*#?&]/);
+		// console.log(results);
```

Comment:
You can remove the commented lines if they are no longer used

Reply:
done